source/manual/howto-manually-remove-assets.html.md

```diff
@@ -14,11 +14,6 @@ follow these steps:
 1. Mark the asset as deleted and remove it from S3
    - [`rake assets:delete_and_remove_from_s3[<asset.id>]`][rake-delete-and-remove-from-s3] (non-Whitehall assets)
    - [`rake assets:whitehall_delete_and_remove_from_s3[<full URL path>]`][whitehall-rake-delete-and-remove-from-s3] (Whitehall assets)
-1. Remove the asset from asset-manager file system (it may not be here as these are automatically removed after S3 upload)
-    1. `gds govuk connect ssh -e production aws/asset_master`
-    1. `cd /mnt/uploads/asset-manager/assets`
-    1. Use the path identified in step 5 to check for the file by removing the `/var/apps/asset-manager/uploads/assets` prefix
-    1. `sudo rm path/to/file`
```

Comment:
Is there a risk that a worker will go ahead and reupload the asset?
(I don't think there is, but just wonder if these instructions were left out of an abundance of caution).

Reply:
No, I don't think so.